File: sdkwork-ai-sdk-python/sdkwork_ai_sdk/models/open_ai_video.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class OpenAiVideo:
    """OpenAI标准视频对象"""
    id: str = None
    object: str = None
    model: str = None
    status: str = None
    progress: int = None
    prompt: str = None
    seconds: str = None
    size: str = None
    quality: str = None
    error: VideoError = None
    created_at: int = None
    completed_at: int = None
    expires_at: int = None
    output_url: str = None
    remixed_from_video_id: str = None

File: sdkwork-ai-sdk-go/types/open_ai_video.go
package types

// OpenAI标准视频对象
type OpenAiVideo struct {
	Id string `json:"id"`
	Object string `json:"object"`
	Model string `json:"model"`
	Status string `json:"status"`
	Progress int `json:"progress"`
	Prompt string `json:"prompt"`
	Seconds string `json:"seconds"`
	Size string `json:"size"`
	Quality string `json:"quality"`
	Error VideoError `json:"error"`
	CreatedAt int `json:"created_at"`
	CompletedAt int `json:"completed_at"`
	ExpiresAt int `json:"expires_at"`
	OutputUrl string `json:"output_url"`
	RemixedFromVideoId string `json:"remixed_from_video_id"`
}
